config/v1/types_node.go

```diff
@@ -0,0 +1,156 @@
+package v1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// Node holds cluster-wide information about node specific features.
+//
+// Compatibility level 1: Stable within a major release for a minimum of 12 months or 3 minor releases (whichever is longer).
+// +openshift:compatibility-gen:level=1
+type Node struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	// spec holds user settable values for configuration
+	// +kubebuilder:validation:Required
+	// +required
+	Spec NodeSpec `json:"spec"`
+
+	// status holds observed values.
+	// +optional
+	Status NodeStatus `json:"status"`
+}
+
+type NodeSpec struct {
+	// CgroupMode determines the cgroups version on the node
+	// +optional
+	CgroupMode CgroupMode `json:"cgroupMode,omitempty"`
```

Comment:
Nit: `CGroupMode` would be more correct

Reply:
Actually, if you're only talking about version, why not just `CGroupVersion`?

Reply:
@rphillips for the comment on cgroups

Reply:
I double checked k8s code, and I've noticed I was looking at comments but vars are using that, sorry for the noise :wink: